datadog_checks_base/datadog_checks/base/utils/aws.py

```diff
@@ -11,13 +11,14 @@ def rds_parse_tags_from_endpoint(endpoint):
     * `dbinstanceidentifier` - The instance's unique identifier for RDS and Aurora clusters
     * `dbclusteridentifier` - The cluster identifier in the case of Aurora cluster endpoints and serverless
     * `hostname` - The full endpoint if the endpoint points to a specific instance
+    * `host` - The full endpoint if the endpoint points to a specific instance
```

Comment:
Small nit: help distinguish `hostname` and `host` description from explanation in https://github.com/DataDog/integrations-core/pull/8292/files#r552717379.

Reply:
I moved the comment up to the docstring to disambiguate the two. Let me know if the descriptions are still unclear without additional context!